index.js

```diff
@@ -1,58 +1,45 @@
-var hyperlog = require('hyperlog')
-var fdstore = require('fd-chunk-store')
 var path = require('path')
 var mkdirp = require('mkdirp')
-var osmdb = require('osm-p2p-db')
-var once = require('once')
-
+var kappa = require('kappa-core')
+var osmdb = require('kappa-osm')
 var LevelUp = require('levelup')
 var leveldown = require('leveldown')
-var deflevel = require('deferred-leveldown')
-var deferredStore = require('deferred-chunk-store')
+var once = require('once')
+var Deflevel = require('deferred-leveldown')
+var raf = require('random-access-file')
 
 module.exports = function (opts) {
   if (typeof opts === 'string') opts = { dir: opts }
   if (!opts) opts = {}
-  var dir = opts.dir || path.resolve('./osm-p2p-data')
-  var defIndexDB = new deflevel()
-  var defLogDB = new deflevel()
+  var dir = opts.dir || path.resolve('./osm-p2p-kappa')
```

Comment:
changed the name to not collide with other databases that could exist, what do you think @noffle ?

Reply:
I'm down with this :+1: